vcpkg/ports/qgis-qt6/vcpkg.json

```diff
@@ -5,6 +5,11 @@
     "description": "QGIS is a free, open source, cross platform (lin/win/mac) geographical information system (GIS)",
     "dependencies": [
       "expat",
+      {
+        "name": "qgis-qt6",
```

Comment:
@m-kuhn , you know while we're at it, we might as well rename qgis-qt6 to qgis in our vcpkg/ports folder.

Reply:
I am afk, but go for it if you want

Reply:
@m-kuhn , OK, I'm momentarily taking over.

Reply:
Thanks. Did you notice the missing `DESTINATION` param in the `file` command too?